schema.sql

```diff
@@ -36,4 +36,14 @@ ALTER TABLE animals
   ADD COLUMN owner_id INT;
 
     ALTER TABLE animals
-    ADD FOREIGN KEY (owner_id) REFERENCES owner(id);
\ No newline at end of file
+    ADD FOREIGN KEY (owner_id) REFERENCES owner(id);
+
+    --Join visits
+    CREATE TABLE vets(id serial PRIMARY KEY,name TEXT, age INTEGER, date_of_graduation DATE);
+
+    CREATE TABLE specializations(species_id INTEGER, vets_id INTEGER, CONSTRAINT FK_species FOREIGN KEY (species_id) REFERENCES species (id), CONSTRAINT FK_vets FOREIGN KEY (vets_id) REFERENCES vets (id) );
+
+
+CREATE TABLE visits(animals_id INTEGER, vets_id INTEGER, CONSTRAINT FK_animals FOREIGN KEY (animals_id) REFERENCES animals (id), CONSTRAINT FK_vets FOREIGN KEY (vets_id) REFERENCES vets (id) );
```

Comment:
- [ ] The only column name that should be changed here if the need be is the `visit_date`. Other than that, you should try to define the column names just as stated per the visits table. `aminals_id` should be `aminal_id` and `vets_id` should be `vet_id`. They should be in the singular form, not plural. Please make this change.

Visits Tabel ERD|
:-----------------------:|
![a1](https://user-images.githubusercontent.com/39523312/178123959-a0a1ebea-b272-45b1-b6f0-4804cc346dda.png)